pkg/daemon/ceph/client/pool.go

```diff
@@ -818,3 +793,27 @@ func getCurrentCrushMap(context *clusterd.Context, clusterInfo *ClusterInfo) (Cr
 
 	return crushMap, nil
 }
+
+// GePoolImages returns all the images associated with a pool
+func GePoolImages(context *clusterd.Context, clusterInfo *ClusterInfo, poolName string) ([]string, error) {
```

Comment:
are we using this helper function somewhere? am not able to see where it is called

Reply:
No. Removed